examples/intersection_observer/src/lib.rs

```diff
@@ -0,0 +1,124 @@
+use seed::{prelude::*, *};
+use web_sys::{IntersectionObserver, IntersectionObserverEntry, IntersectionObserverInit};
+
+fn init(_: Url, orders: &mut impl Orders<Msg>) -> Model {
+    orders.after_next_render(|_| Msg::SetupObserver);
+    Model {
+        box_container: ElRef::new(),
+        red_box: ElRef::new(),
+        observer: None,
+        observer_callback: None,
+        observer_entries: None,
+    }
+}
+
+struct Model {
+    box_container: ElRef<web_sys::Element>,
+    red_box: ElRef<web_sys::Element>,
+    observer: Option<IntersectionObserver>,
+    observer_callback: Option<Closure<dyn Fn(Vec<JsValue>)>>,
+    observer_entries: Option<Vec<IntersectionObserverEntry>>,
+}
+
+enum Msg {
+    SetupObserver,
+    Observed(Vec<IntersectionObserverEntry>),
+}
+
+fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
+    match msg {
+        Msg::Observed(entries) => {
+            model.observer_entries = Some(entries);
+        }
+        Msg::SetupObserver => {
+            orders.skip();
+
+            // ---- observer callback ----
+            let sender = orders.msg_sender();
+            let callback = move |entries: Vec<JsValue>| {
+                let entries = entries
+                    .into_iter()
+                    .map(IntersectionObserverEntry::from)
+                    .collect();
+                sender(Some(Msg::Observed(entries)));
+            };
+            let callback = Closure::wrap(Box::new(callback) as Box<dyn Fn(Vec<JsValue>)>);
+
+            // ---- observer options ----
+            let mut options = IntersectionObserverInit::new();
+            options.threshold(&JsValue::from(1));
+            // ---- observer ----
+            let observer =
+                IntersectionObserver::new_with_options(callback.as_ref().unchecked_ref(), &options)
+                    .unwrap();
+
+            observer.observe(&model.red_box.get().unwrap());
+
+            // Note: Drop `observer` is not enough. We have to call `observer.disconnect()`.
+            model.observer = Some(observer);
+            model.observer_callback = Some(callback);
+        }
+    }
+}
+
+fn view(model: &Model) -> Node<Msg> {
+    div![
+        view_info(model.observer_entries.as_ref()),
+        view_box_container(model),
+    ]
+}
+
+fn view_info(observer_entries: Option<&Vec<IntersectionObserverEntry>>) -> Option<Node<Msg>> {
+    let entry = observer_entries?.first()?;
+    Some(div![
+        style! {
+            St::Position => "fixed",
+            St::Background => "white",
+        },
+        div![format!("Target: {:#?}", entry.target().id()),],
+        div![format!(
+            "Is intersecting entirely: {:#?}",
```

Comment:
```suggestion
        div![format!("Target: {}", entry.target().id()),],
        div![format!(
            "Is intersecting entirely: {}",
```
I think `ToString` is enough.